tests/integration/s2n_client_endpoint_handshake_test.py

```diff
@@ -125,6 +125,8 @@ def well_known_endpoints_test(use_corked_io, tls13_enabled):
     for endpoint_config in well_known_endpoints:
 
         endpoint = endpoint_config["endpoint"]
+        if endpoint in allowed_endpoints_failures and tls13_enabled:
+            continue
```

Comment:
Why is this necessary? Doesn't just adding the endpoint to the `allowed_endpoints_failures` array suffice:

https://github.com/awslabs/s2n/blob/89fa7ecc41fcd37ee12a94f7bfe310e08c3213d9/tests/integration/s2n_client_endpoint_handshake_test.py#L144-L145

Reply:
If I only add the yahoo to the allowed_endpoints_failures array, we'll end up in the retry logic, and with the max retry limit of 5, it takes a long time for the tests to fail. It's faster to just skip over the endpoints we know are going to fail. It does make line 144 redundant though 🤷.

Reply:
Previously (I don't know how far back), wikipedia was only failing part of the time. It's failing consistently now for a different reason so I think it's better to just skip it entirely right now.